### PR TITLE
Features/tweets

### PR DESCRIPTION
**Pull request by Draqe**
@nzdrml

### CODE REVIEW

File: app/controllers/tweets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  end
+
+  def destroy
+    @tweet = Tweet.find(params[:id])
+    @tweet.destroy
+    redirect_to tweets_path
+  end
+    
+  private
+    def tweet_params
```
Comment (nzdrml) [inline app/controllers/tweets_controller.rb]: convention for this is the `def` block is the same indentation level as `private`

Add one blank line in between

File: app/controllers/tweets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    @tweet = Tweet.new
+  end
+
+  def show
+    @tweet = Tweet.find(params[:id])
+  end
+
+  def create
+    @tweet = Tweet.new(tweet_params)
+      if @tweet.save
```
Comment (nzdrml) [inline app/controllers/tweets_controller.rb]: why is this indented

File: app/controllers/tweets_controller.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      end
+  end
+
+  def edit
+    @tweet = Tweet.find(params[:id])
+  end
+
+  def update
+    @tweet = Tweet.find(params[:id])
+      if @tweet.update(tweet_params)
```
Comment (nzdrml) [inline app/controllers/tweets_controller.rb]: whiy is this indented

**Comment by Draqe** [comment]
@nzdrml

**Pull request merged**